### DISCUSSION TITLE
最初的全角色版面

### DISCUSSION THREAD

**Discussion by Oyuanchu** [Arondight/Adachi-BOT#802] [Q&A] [answered]
哪里有最初的版面的模板数据嘞

**Comment by mark9804** [comment]
没有理解，麻烦描述的清楚一些

**Comment by Arondight** [comment, marked as answer]
1. https://github.com/Arondight/Adachi-BOT/tree/a65608bdb2b9c9cb2d704c94b434ec0e373a9032/src/plugins/card
2. https://github.com/Arondight/Adachi-BOT/blob/a65608bdb2b9c9cb2d704c94b434ec0e373a9032/src/utils/render.js
3. https://github.com/Arondight/Adachi-BOT/blob/a65608bdb2b9c9cb2d704c94b434ec0e373a9032/src/views/genshin-card.html
4. https://github.com/Arondight/Adachi-BOT/tree/a65608bdb2b9c9cb2d704c94b434ec0e373a9032/src/views/component/card

图片看 HTML

**Comment by Oyuanchu** [comment]
谢谢，佬
